src/routes/acquirerRoutes.ts

```diff
@@ -42,6 +42,9 @@ const skipThisStep = (step: string, formdata: FormData) => {
       // Skip other-orgs if the answer to data-access was "no"
       return formdata.steps["data-access"].value === "no";
     }
+    case "role": {
+      return formdata.steps["data-type"].value === "none";
```

Comment:
Thanks @gabimartini. I think we also need to skip the `role` step if the `data-type` step has not been answered (so it would still have its default value). Maybe something like:
```suggestion
      return formdata.steps["data-type"].value === "none" || formdata.steps["data-type"].value === "";
```